packages/api/package.json

```
@@ -9,22 +9,25 @@
     "lt": "npm-run-all -p lt:*",
     "lt:cluster": "npx localtunnel --port 9094 --subdomain \"$(whoami)-cluster-api-web3-storage\"",
     "lt:postgres": "npx localtunnel --port 3000 --subdomain \"$(whoami)-postgres-api-web3-storage\"",
-    "start": "npm-run-all -p cluster:start pg:start -s miniflare",
-    "start:preview": "npm-run-all -p cluster:start pg:start -s -p lt  wrangler-dev",
-    "stop": "npm-run-all -p cluster:stop pg:stop",
+    "start": "npm-run-all -p minio:start \"minio:mkbucket -- dotstorage-dev-0\" cluster:start pg:start -s miniflare",
+    "start:preview": "npm-run-all -s minio:start \"minio:mkbucket -- dotstorage-dev-0\" -p cluster:start pg:start -s -p lt  wrangler-dev",
+    "stop": "npm-run-all -p minio:stop cluster:stop pg:stop",
     "wrangler-dev": "wrangler dev --env $(whoami)",
     "publish": "wrangler publish --env $(whoami)",
     "build": "node scripts/cli.js build",
-    "test": "npm-run-all -p -r mock:backup test:worker -s test:size",
-    "test:worker": "mocha --require test/hooks.js --experimental-vm-modules --timeout 5000",
+    "test": "run-s test:worker test:size",
+    "test:worker": "mocha --require test/hooks.js --experimental-vm-modules --timeout 5000 --bail",
```

This looks like it might be left over from testing out tests.
```suggestion
    "test:worker": "mocha --require test/hooks.js --experimental-vm-modules --timeout 5000",
```